src/main.ts

```diff
@@ -9,141 +7,189 @@ import generateSeqD from "./generateSeqD"
 import generateSD from "./generateSD"
 import generateTS from "./codeGen"
 import validateSeqD from "./validateSeqD"
+const inquirer = require("inquirer");
+
+const questions = [
+    {
+        type: "list",
+        name: "generationType",
+        message: "What's your desired generation type?",
+        choices: ["Sequence Diagram -> System Description", "System Description -> Sequence Diagram"],
+        filter(val) {
+            if (val === "Sequence Diagram -> System Description") {
+                return 0;
+            }
+            return 1;
+        },
+    },
+    {
+        type: "input",
+        name: "SeqDPath",
+        message: "Path to folder containing your Sequence Diagram(s) (relative to this directory)",
+        default: "./example-input/SeqD-examples/SeqDs",
+        when(answers) {
+            return answers.generationType === 0;
+        },
+    },
+    {
+        type: "input",
+        name: "TDsPath",
+        message: "Path to Thing Descriptions (relative to this directory)",
+        default: "./example-input/SeqD-examples/TDs.json",
+        when(answers) {
+            return answers.generationType === 0;
+        },
+    },
+    {
+        type: "input",
+        name: "SDPath",
+        message: "Path to folder containing your System Description(s) (relative to this directory)",
+        default: "./example-input/SD-examples/SDs",
+        when(answers) {
+            return answers.generationType === 1;
+        },
+    },
+    {
+        type: "input",
+        name: "outputPath",
+        message: "Where to put created output? (relative to this directory)",
+        default: "./created-output",
+    },
+];
+
+inquirer.prompt(questions).then((answers) => {
```

Comment:
Can you do this with await in order to reduce apparent changes?

Reply:
I am not sure this will reduce the number of changes since I will still need to wrap everything inside another async function in order to do `await inquirer.prompt(questions)`.